File: 0283-move-zeroes/0283-move-zeroes.java
class Solution {
    public void moveZeroes(int[] nums) {
        int n = nums.length;
        int i=0;
        int j=0;  //handling zero
        while(i<n){
            if(nums[i]!=0){
                int temp = nums[j];
                nums[j]=nums[i];
                nums[i]=temp;
                j++;
            }
            i++;

        }
        
    }
}

File: 0283-move-zeroes/0283-move-zeroes.py
class Solution(object):
    def moveZeroes(self, nums):
        """
        :type nums: List[int]
        :rtype: None Do not return anything, modify nums in-place instead.
        """
        i = 0
        j = 0
        n = len(nums)
        while i < n:
            if nums[i] != 0:
                temp = nums[i]
                nums[i] = nums[j]
                nums[j] = temp
                j+=1
            i+=1

        
        